knowledge-base/bid-decay-mechanism.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Bid decay mechanism
+sidebarTitle: Bid Decay
+description: "Learn about how bids decay in value after they are issued."
+---
+
+In the mev-commit auction for transaction bundles, bidders submit their bids to providers and their goal is to receive a confirmation that the bundle they bid for will be included in the next block produced by the provider. Bidders aim to receive confirmation for their bundle as soon as possible since early confirmations tend to be more valuable to the bidders. However, providers may want to wait until the last moment to issue confirmations in order to build the most valuable block possible without missing out on any late bids.
+
+Therefore, it's essential to have a mechanism that encourages providers to confirm bids as early as possible. This is where a bid decay mechanism comes into play, which reduces the value of a bid based on the time elapsed since it was issued.
+
+## Mechanism Description:
+
+Every bid has two timestamps attached to it: (1) a timestamp indicating the exact time of dispatch of the bid, and (2) an expiry timestamp, marking the moment when the decay for this bid reaches 100%—essentially, the point in time beyond which the bidder is unwilling to pay anything to a provider for a commitment. 
+
+When a provider decides to commit to the bid, they must issue a commitment and also assign a "commitment timestamp" to it. This commitment is then sent to the mev-commit chain for inclusion. The mev-commit chain will include the commitment into the chain, however the commitment will only be considered valid if the commitment timestamp supplied by the provider and the block timestamp where the commitment was included are "close enough", or more formally, are within some time interval $\Delta$.
+
+The parameter $\Delta$ is set to be $250$ms, and its goal is to account for the latency in message delivery among nodes and the discrepancy between the commitment and block timestamps. More precisely, for validation of a commitment it is required that $t_b - t_c \le \Delta$, where $t_c$ denotes the commitment timestamp and $t_b$ denotes the timestamp of the block where the commitment was included.
```

Comment:
It seems it will be \Delta = 350 ms.

Reply:
I can update, but shouldn't we get some "real" number for this before we add anything to the docs on a concrete value?